test/e2e/storage/volume_metrics.go

```diff
@@ -52,8 +52,12 @@ var _ = utils.SIGDescribe("[Serial] Volume metrics", func() {
 	BeforeEach(func() {
 		c = f.ClientSet
 		ns = f.Namespace.Name
+		var err error
 		framework.SkipUnlessProviderIs("gce", "gke", "aws")
-		defaultScName = getDefaultStorageClassName(c)
+		defaultScName, _ = framework.GetDefaultStorageClassName(c)
+		if err != nil {
+			framework.Failf(err.Error())
```

Comment:
Should this be a skip? I think we should consider removing the skip based on provider

Reply:
Since some of these tests depend on underlying volume plugin implementing the particular metric interface, it may not be sufficient for default storageclass to be found.